### PR TITLE
Add initial config for e2e and smoke test

### PR DESCRIPTION
**Pull request by tortmayr**
Signed-off-by: Eugen Neufeld <eneufeld@eclipsesource.com>

### CODE REVIEW

**Comment by tortmayr** [comment]
@eneufeld  The new stage is now picked up by the CI but it seems like there are some configuration errors.

File: client/configs/base.tsconfig.json [hunk 1]
```diff
@@ -1,25 +1,30 @@
...
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "target": "es5",
+    "jsx": "react",
+    "lib": [
+      "es6",
+      "dom"
+    ],
+    "sourceMap": true,
+    "types": [
```
Comment (eneufeld) [inline client/configs/base.tsconfig.json:24]:
```suggestion
```

File: client/configs/base.tsconfig.json [hunk 2]
```diff
@@ -1,25 +1,30 @@
...
+      "node",
```
Comment (eneufeld) [inline client/configs/base.tsconfig.json:25]:
```suggestion
```

File: client/configs/base.tsconfig.json [hunk 3]
```diff
@@ -1,25 +1,30 @@
...
+      "@wdio/sync",
```
Comment (eneufeld) [inline client/configs/base.tsconfig.json:26]:
```suggestion
```

File: client/configs/base.tsconfig.json [hunk 4]
```diff
@@ -1,25 +1,30 @@
...
+      "mocha"
```
Comment (eneufeld) [inline client/configs/base.tsconfig.json:27]:
```suggestion
```

File: client/configs/base.tsconfig.json [hunk 5]
```diff
@@ -1,25 +1,30 @@
...
+    ]
```
Comment (eneufeld) [inline client/configs/base.tsconfig.json:28]:
```suggestion
```

File: client/configs/base.tsconfig.json [hunk 6]
```diff
@@ -1,25 +1,25 @@
...
+    "downlevelIteration": true,
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "target": "es5",
+    "jsx": "react",
+    "lib": [
+      "es6",
+      "dom"
+    ],
+    "sourceMap": true
```
Comment (eneufeld) [inline client/configs/base.tsconfig.json:23]:
```suggestion
    "sourceMap": true,
    "types": [
      "node",
      "@wdio/sync",
      "@wdio/mocha-framework",
      "mocha"
    ]
```

File: client/configs/wdio.config.ts [hunk 7]
```diff
@@ -0,0 +1,367 @@
...
+                    return `wdio-junit-report-${options.capabilities.browserName}-${options.cid}.xml`;
+                }
+            }]
+        ],
+
+        //
+        // Options to be passed to Mocha.
+        // See the full list at http://mochajs.org/
+        mochaOpts: {
+            ui: 'bdd',
```
Comment (eneufeld) [inline client/configs/wdio.config.ts:181]:
```suggestion
```

**Pull request closed without merging**